### PR TITLE
Add publicfeeds.net CDN template

### PR DESCRIPTION
**Pull request by farski**
Closes https://github.com/PRX/internal/issues/872

### CODE REVIEW

File: cdn/publicfeeds-cdn.yml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      Runtime: nodejs16.x
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Feed CDN }
+      Timeout: 1
+  OriginRequestFunctionVersionA:
```
Comment (cavis) [inline cdn/publicfeeds-cdn.yml:94]: Does this auto-update when the code changes?  Or do you have to cycle this?

File: cdn/publicfeeds-cdn.yml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        Name: !Sub ${AWS::StackName}-origin-req-policy
+        QueryStringsConfig:
+          QueryStringBehavior: none
+          # QueryStrings:
+
+  CloudFrontDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Aliases: !If [IsProduction, publicfeeds.net, staging.publicfeeds.net]
```
Comment (cavis) [inline cdn/publicfeeds-cdn.yml:139]: Could also make these domains configurable?  Though I doubt we'd ever set this up anywhere else.

**Review by cavis — APPROVED** [review]
👍 looks good to me

File: cdn/publicfeeds-cdn.yml (hunk 1 above)
Comment (farski) [reply cdn/publicfeeds-cdn.yml:94]: Have to change the resource name

**Pull request merged**